tests/php/api/endpoint/class-test-report.php

```diff
@@ -89,7 +89,7 @@ public function setUp() {
 	 * @covers ::register_routes()
 	 */
 	public function test_register_routes() {
-		$routes = $this->server->get_routes();
+		$routes = rest_get_server()->get_routes();
```

Comment:
This should be unchanged and then on line 35 in the setUp method it should be:
```
$this->server = rest_get_server();
```
Which also means removing the global and action which is handled in that function.

Reply:
The reason I'm using the helper function here (and the other replacements) is because the REST server will be cached and readily available through it. Also, its supposed to offer better [memory cycling](https://github.com/WordPress/wordpress-develop/commit/9e31509293e93383a97ed170039dafa5042ea286).